src/components/Wizards/CreateVmWizard/NetworksTab.js

```diff
@@ -23,6 +23,22 @@ const validateNetwork = network => {
   return errors;
 };
 
+export const validateNetworksNamespace = (networkConfigs, namespace, networks) => {
+  const availableNetworkConfigs = networkConfigs.filter(nc => nc.metadata.namespace === namespace);
+  networks.filter(network => isBootableNetwork(network)).forEach(network => {
+    if (!network.errors) {
+      network.errors = Array(4).fill(null);
```

Comment:
these numeric constants and hard-coded indexes will become nightmare soon

Reply:
I agree. I will refactor such cases as a part of https://github.com/kubevirt/web-ui-components/issues/77